components/button-cards/card.js

```diff
@@ -0,0 +1,64 @@
+export function setCardContentMaximumHeight(rowSelector, element) {
+  const cardRows = document.querySelectorAll(rowSelector)
+
+  for (let i = 0; i < cardRows.length; i++) {
+    const cardRow = cardRows[i]
+    const tallestContent = getContentHeights(element, cardRow)
+
+    setContentHeight(element, tallestContent)
+  }
+}
+
+export function resetCardContentHeights(selector) {
+  const itemContentHeadings = document.querySelectorAll(selector)
+  itemContentHeadings.forEach(function(item) {
+    item.style.height = ''
+  })
+}
+
+function getContentHeights(selector, cardRow) {
+  const contents = cardRow.querySelectorAll(selector)
+  const contentHeights = []
+  const rowMaxHeight = [0]
+  let currentRowIndex = 0
+  let currentOffsetPosition
+
+  for (let i = 0; i < contents.length; i++) {
+    const content = contents[i]
```

Comment:
Content, is that all content or is it a heading or a description or something completely different? :)

Reply:
its for any content so it is head or description